test/e2e/framework/framework.go

```diff
@@ -149,7 +149,9 @@ func (f *Framework) DeleteEtcdOperatorCompletely() error {
 	if err != nil {
 		return err
 	}
-	err = retryutil.Retry(2*time.Second, 5, func() (bool, error) {
+	// On k8s 1.6.1, grace period isn't accurate. It took ~10s for operator pod to completely disappear.
```

Comment:
is it easy to reproduce without our operator?

probably we should create an issue on upstream if we are sure it is a k8s issue.

but i am fine with this quick fix to stabilize our ci.